app/dash_app.py

```diff
@@ -43,17 +43,22 @@
 app = Dash(__name__,
            prevent_initial_callbacks=True,
            external_stylesheets=[dbc.themes.DARKLY],
-           use_pages=True,)
-# app = DashProxy(prevent_initial_callbacks=True,
-#                 suppress_callback_exceptions=True,
-#                 transforms=[MultiplexerTransform()],
-#                 external_stylesheets=[dbc.themes.DARKLY],
-#                 use_pages=True)
+           use_pages=True,
+           suppress_callback_exceptions=True)
+server = app.server
 
 app.layout = html.Div([
-  dash.page_container
-])
+  dash.page_container])
+
+
+def register_routes(server):
+    """ Registers routes for dynamic signed URLs."""
+    from routes import add_routes
```

Comment:
Import statement should go at the top of the file with other imports

Reply:
Fixed in [42632a4](https://github.com/Volcano-Risk-Reduction-in-Canada/Volcanic-Interpretation-Workbench/pull/68/commits/42632a4a846db7eac49a02cf65773032f63d9ba5)